File: family_order/src/web/com/asiainfo/veris/crm/order/web/person/interboss/authoperation/AuthDelayTest.java
package com.asiainfo.veris.crm.order.web.person.interboss.authoperation;

import org.apache.tapestry.IRequestCycle;

import com.ailk.common.data.IData;
import com.ailk.common.data.impl.DataMap;
import com.asiainfo.veris.crm.order.web.frame.bcf.base.CSViewCall;
import com.asiainfo.veris.crm.order.web.frame.csview.person.base.PersonBasePage;

public abstract class AuthDelayTest extends PersonBasePage{
	
	public void onTradeSubmit(IRequestCycle cycle) throws Exception
    {
        IData pageData = getData();
        IData param = new DataMap();
        param.put("ID_VALUE", pageData.getString("ID_VALUE"));
        param.put("IDENT_CODE", pageData.getString("IDENT_CODE"));
        param.put("EFFECTIVE_TIME", pageData.getString("EFFECTIVE_TIME"));
        IData data = CSViewCall.callone(this, "SS.AuthDelayTestSVC.authDelay", param);
        setInfo(data);
        setAjax(data);
    }
	
	public abstract void setInfo(IData info);
}

File: family_order/html/person/interboss/authoperation/scripts/authdelaytest.js
function submitReqParam()
{
	
	var param = "&ID_VALUE=" + $("#ID_VALUE").val()
				+"&IDENT_CODE=" + $("#IDENT_CODE").val()
				+"&EFFECTIVE_TIME=" + $("#EFFECTIVE_TIME").val();
	$.ajax.submit(null, 'onTradeSubmit', param, '', function(data)
	{
		if("0000" == data.get("BIZ_ORDER_RESULT")){
			alert("延时成功！");
		} else {
			alert("延时失败！归属省返回错误：" + data.get("RESULT_DESC"));
		}
	},	
	function(error_code,error_info)
	{
		$.endPageLoading();
		alert(error_info);
	});
}